spec/requests/authentication_spec.rb

```diff
@@ -104,7 +104,10 @@
         "group_href" => "/api/groups/#{group2.id}",
         "role"       => @role.name,
         "role_href"  => "/api/roles/#{group2.miq_user_role.id}",
-        "tenant"     => @group.tenant.name
+        "tenant"     => @group.tenant.name,
+        "miq_groups" => a_collection_including(
```

Comment:
Thanks @jntullo for the update.  Maybe we should also add the groups =>  [ ] here before miq_groups for completion.  It may never have been added to the spec.